apple/versioning.bzl

```diff
@@ -254,8 +254,8 @@ apple_bundle_version(
     build_label_pattern = "MyApp_{version}_build_{build}",
     build_version = "{version}.{build}",
     capture_groups = {
-        "version": "\\d+\\.\\d+",
-        "build": "\\d+",
+        "version": "\\\\d+\\\\.\\\\d+",
+        "build": "\\\\d+",
```

Comment:
This change isn't really useful for reading the .bzl file, but because this gets used to generate the doc, *each* backslash needs to be double-escaped.